core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -150,11 +151,16 @@ protected void configure() {
         .annotatedWith(BFTSyncPatienceMillis.class)
         .to(properties.get("bft.sync.patience", 200));
 
-    // Max timeout = (1.2^8)×3 ~= 13s
-    bindConstant().annotatedWith(PacemakerBaseTimeoutMs.class).to(3000L);
-    bindConstant().annotatedWith(PacemakerBackoffRate.class).to(1.2);
-    bindConstant().annotatedWith(PacemakerMaxExponent.class).to(8);
-    bindConstant().annotatedWith(AdditionalRoundTimeIfProposalReceivedMs.class).to(30_000L);
+    /* Default timeouts config:
+    Max exponential timeout (based on consecutive timeout occurrences) = (1.2^8)×3 ~= 13s
+    Additionally, when vertex store reaches 2/3 of its max capacity (that is: 2/3*150 = 100 MB by default)
+    we start multiplying the timeout by a linearly increasing value, up to 10x.
+    So a maximum theoretical timeout is 130s. */
+    bind(PacemakerTimeoutCalculatorConfig.class)
+        .toInstance(new PacemakerTimeoutCalculatorConfig(3000L, 1.2, 8, 30_000L, 0.66, 10));
+
+    // Delayed resolution is disabled for now.
```

Comment:
Might it be worth explaining why in this comment? (i.e. because we cannot create QCs on fallback vertices, because we don't just sign the ledger header, but also the BFT header, which captures the previous certificate chain, and all the nodes have a different certificate chain for their fallback vertex)